api/server.py

```diff
@@ -199,7 +199,6 @@ def authorization_check():
             abort(403, "unauthorized IP")
     if not api_access_without_key:
         is_authorized()
-    return
```

Comment:
@HarshCasper : thank you for the PR.
I think the return should be there, as it is explicit that the function returns None, otherwise also it would work , but then one must know how python works internally and python treats it as returning None.

Reply:
@HarshCasper : I do not think we need to remove return from functions.